tests/fixtures/test_alfasim_runner.py

```diff
@@ -70,3 +70,29 @@ def test_alfasim_runner_add_plugin(
     results = alfasim_runner.run()
     assert alfasim_runner.alfacase_file.is_file()
     assert len(results.list_profiles()) == 2
+
+
+def test_alfasim_runner_from_case_description(
+    alfasim_runner: AlfasimRunnerFixture,
+    fake_alfasim: None,
+    empty_alfacase: Path,
+    abx_plugin_source: Path,
+) -> None:
+    from alfasim_sdk import convert_alfacase_to_description
+
+    case_description = convert_alfacase_to_description(Path(empty_alfacase))
```

Comment:
does `convert_alfacase_to_description` returns a list?

Reply:
No, it returns a `CaseDescription` object.